birdhouse/pavics-compose.sh

```diff
@@ -106,6 +113,12 @@ done
 # components.
 [ -f env.local ] && . ./env.local
 
+for i in ${DELAYED_EVAL}; do
+  v="`eval "echo \\$${i}"`"
+  eval 'export ${i}="`eval "echo ${v}"`"'
+  echo "delayed eval '`env |grep ${i}=`'"
+done
```

Comment:
I'm not sure if I understand the need for the separate `DELAYED_EVAL` definition. 
Wouldn't developers forgetting to specify a new variable name in it be more prone to error? 
Couldn't all variables in `${VARS}` could do the ```v="`eval "echo \\$${i}"`"``` step, and this way simply using the single quotes would be sufficient to achieve the same result?

Reply:
Well, if you think for simplicity, all vars in `default.env` should be delayed eval then `VARS` do not contain all of them.  `VARS` currently do not contain the 3 broken vars in `DELAYED_EVAL`.

Then it looks like delayed eval is an exception rather than the norm.  So I'd rather not forced the single quote on more vars than needed.

Reply:
Ouch, the grep trick might too magic for me.  How do you get the name of each var after?

Reply:
Using this:
 cat birdhouse/default.env | grep -E "export [A-Z0-9_]+=" | cut -d ' ' -f 2 | cut -d '=' -f 1`

Reply:
And an extra ` | sed -r -e 's/(.*)/\$\1/g'` at the end for adding the `$`.

Reply:
Oh wow, think I'll pass on this magic for now.  We have to handle all the `default.env` from components from any other external repos as well.  I don't have time to think of all the corner cases with this magic for now.

Do you see anything blocking with the current approach?

Reply:
No. Good to go with the current alternative.